packages/zod-form-data/README.md

```diff
@@ -163,6 +163,35 @@ the `FormData` will not include an entry for `myCheckbox` at all.
 
 ([Further reading](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value))
 
+### file
+
+Transforms any empty File objects to `undefined` before validating.
+This makes it so empty files will fail required checks,
+allowing you to use `optional` for optional fields.
+If you call `zfd.file` with no arguments, it will assume the field is a required file by default.
+
+#### Usage
+
+```ts
+const schema = zfd.formData({
+  requiredFile: zfd.file(),
+  optional: zfd.file().optional(),
```

Comment:
I can make this change after I merge, but want to give you a heads up since you're probably going to use this. Doing `zfd.file().optional()` doesn't actually work since because the `optional` gets evaluated before the `preprocess`. So the way to make it optional is actually `zfd.file(z.instanceOf(File).optional())`.

Reply:
I did not know this O.o seems so weird that there's no workaround for that scenario